### PR TITLE
#166726242 buyer should be able to update purchase order

### PR DESCRIPTION
**Pull request by breezycloud**
#### What does this PR do?
PATCH request: update purchase order

#### Description of Task to be completed
This feature should allow the buyer to update purchase order

#### How should this be manually tested?
This feature can be tested using POSTMAN

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166726242 ](https://www.pivotaltracker.com/story/show/166726242)

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: API/test/order.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var expect  = require('chai').expect;
```
Comment (TheoOkafor) [inline API/test/order.js]: Use ES6 syntax

**Review by TheoOkafor — CHANGES_REQUESTED** [review]
The build is failing on Travis. Make sure that build passes before you merge

File: API/test/order.js (hunk 1 above)
Comment (breezycloud) [reply API/test/order.js]: Fixed.

**Review by TheoOkafor — APPROVED** [review]
Good!

**Pull request merged**